Rawl_B502/de_raind_h_érenn.trig

```diff
@@ -28,7 +28,7 @@
 <#ÉoganMár>
     a foaf:Person ;
     irishRel:nomName "Éogan Már" ;
-    owl:sameAs <http://example.com/LL/eoganachta_casil.trig#EogainMóir>.
+    owl:sameAs <http://example.com/Rawl_B502/de_genelogia_síl_ébir.trig#MugNuadat>.
```

Comment:
This kind of goes with the comment above is Eogan Táidlech the same as Éogan Már and Mug Naudat?

Reply:
Well-spotted. This is one is incorrect.